### DISCUSSION TITLE
Code toggle for any language

### DISCUSSION THREAD

**Discussion by nunomazer** [razonyang/hugo-theme-bootstrap#860] [Q&A] [answered]
Hi, greate theme.

I using it for some months already but struggle to make a code toggle to other languages/formats than toml, yaml.

I wanted to build a widget where I can write an example of code in  several languages to be viewed as a toggle, like sql, javascript, php

Is there something I'm missing on docs or I should build this kind of widget myself?

**Comment by razonyang** [comment]
Thanks for using this theme.

You'll need to build your own yourself, since the `code-toggle` shortcode generates `JSON`, `YAML` and `TOML` code block from one code base (one of those three), it couldn't be used on the languages you mentioned.

It makes sense that create a universal toggle shortcode for stuff like this, I'll create a PR for this if it's possible.

**Comment by razonyang** [comment, marked as answer]
Now this feature is available since [25591d0](https://github.com/razonyang/hugo-theme-bootstrap/commit/25591d0ce8a52862059086ae6590a7a5915ad6de). You'll need to upgrade the theme to that commit or the `master` branch.

See [Toggle](https://hbs.razonyang.com/v1/en/docs/shortcodes/toggle/) for details.

**Reply by nunomazer** [reply]
Man you are amazing, worked like a charm, thank you so much for the support.
